composer.json

```diff
@@ -4,7 +4,7 @@
     "keywords": ["php-pusher-server", "pusher", "rest", "realtime", "real-time", "real time", "messaging", "push", "trigger", "publish", "events"],
     "license": "MIT",
     "require": {
-        "php": ">=7.1 <7.4",
+        "php": ">=7.1 <7.5",
```

Comment:
Shouldn't this be `>=7.1 <7.5` as the next release after `7.4` would be `8`.

Reply:
7.5 is fine IMO. Better to be clear that it only works on 7.1, 7.2, 7.3, and 7.4.

Reply:
Or, tbh, it's more common to just do `^7.1` these days.

Reply:
Yes I think that is reasonable since minor versions are backwards compatible.

Reply:
> Yes I think that is reasonable since minor versions are backwards compatible.

Actually, they are not. PHP makes breaking changes in minor releases, every time.

Reply:
Hmm. According to https://wiki.php.net/rfc/releaseprocess, they shouldn't be!